TrainingGym/Scenarios/VitalTri/VitalTriAM/VitalTriTrimAM.net.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- generated on Sat 23 Jan 2021 01:31:22 PM by Eclipse SUMO netedit Version 1.7.0
+<!-- generated on 01/30/21 17:44:29 by Eclipse SUMO netedit Version 1.7.0
 <configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/netconvertConfiguration.xsd">
 
     <input>
-        <sumo-net-file value="/home/eric/Github/G15_Capstone/TrainingGym/Scenarios/VitalTri/VitalTriAM/VitalTriTrimAM.net.xml"/>
+        <sumo-net-file value="C:\Users\cau\Documents\GitHub\G15_Capstone\TrainingGym\Scenarios\VitalTri\VitalTriAM\VitalTriTrimAM.net.xml"/>
     </input>
 
     <output>
-        <output-file value="/home/eric/Github/G15_Capstone/TrainingGym/Scenarios/VitalTri/VitalTriAM/VitalTriTrimAM.net.xml"/>
+        <output-file value="C:\Users\cau\Documents\GitHub\G15_Capstone\TrainingGym\Scenarios\VitalTri\VitalTriAM\VitalTriTrimAM.net.xml"/>
```

Comment:
This isn't something that needs to change, but I just thought it was entertaining to see all the slashes change when the file gets saved on windows vs unix